src/fr/phenixscans/src/eu/kanade/tachiyomi/extension/fr/phenixscans/PhenixScans.kt

```diff
@@ -1,24 +1,304 @@
+                title = it.title
+                thumbnail_url = "$apiBaseUrl/${it.coverImage}" // Possibility of using ?width=75
+                url = "$baseUrl/manga/${it.slug}"
+            }
+        }
+
+        return MangasPage(mangas, false)
+    }
+
+    // =============================== Latest ===============================
+    override fun latestUpdatesRequest(page: Int): Request {
+        val url = apiBaseUrl.toHttpUrl().newBuilder()
+            .addPathSegment("front").addPathSegment("homepage")
+            .addQueryParameter("page", page.toString())
+            .addQueryParameter("section", "latest")
+            .addQueryParameter("limit", "12")
+            .build()
+
+        Log.e("PhenixScans", url.toString())
+
+        return GET(url.toString(), headers)
+    }
+
+    private fun parseMangaList(mangaList: List<LatestManga>): List<SManga> {
+        return mangaList.map {
+            SManga.create().apply {
+                title = it.title
+                thumbnail_url = "$apiBaseUrl/${it.coverImage}" // Possibility of using ?width=75
+                url = "$baseUrl/manga/${it.slug}"
+            }
+        }
+    }
+
+    override fun latestUpdatesParse(response: Response): MangasPage {
+        val data = response.parseAs<LatestMangaResponse>()
+
+        val mangas = parseMangaList(data.latest)
+
+        val hasNextPage = data.pagination.currentPage < data.pagination.totalPages
+
+        return MangasPage(mangas, hasNextPage)
+    }
+
+    // =============================== Search ===============================
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        if (query.isNotEmpty()) {
+            // No limits here
+            val apiUrl = "$apiBaseUrl/front/manga/search".toHttpUrl().newBuilder()
+                .addQueryParameter("query", query)
+                .build()
+            return GET(apiUrl, headers)
+        }
+
+        val url = "$apiBaseUrl/front/manga".toHttpUrl().newBuilder()
+        filters.forEach { filter ->
+            when (filter) {
+                is SortFilter -> {
+                    url.addQueryParameter("sort", filter.toUriPart())
+                }
+                is GenreFilter -> {
+                    val genres = filter.state
+                        .filter { it.state }
+                        .map { it.id }
+
+                    url.addQueryParameter("genre", genres.joinToString(","))
+                }
+                is TypeFilter -> {
+                    url.addQueryParameter("type", filter.toUriPart())
+                }
+                is StatusFilter -> {
+                    url.addQueryParameter("status", filter.toUriPart())
+                }
+                else -> {}
+            }
+        }
+        url.addQueryParameter("limit", "18") // Be cool on the API
+        url.addQueryParameter("page", page.toString())
+
+        Log.e("PhenixScans", "Search url: ${url.build()}")
```

Comment:
remove logs

Reply:
This is planned before I put the PR in a non-draft state